website/content/en/preview/upgrade/upgrade-ref.md

```diff
@@ -0,0 +1,606 @@
+
+| StatusCondition Types           |                |
+|---------------------------------|----------------|
+| **v1alpha5**                    | **v1beta1**    |
+| MachineLaunched                 | Launched       |
+| MachineRegistered               | Registered     |
+| MachineInitialized              | Initialized    |
+| MachineEmpty                    | Empty          |
+| MachineExpired                  | Expired        |
+| MachineDrifted                  | Drifted        |
+
+## Provisioner to NodePool
+
+Karpenter v1beta1 moves almost all top-level fields under the `NodePool` template field. Similar to Deployments (which template Pods that are orchestrated by the deployment controller), Karpenter NodePool templates NodeClaims (that are orchestrated by the Karpenter controller). Here is an example of a `Provisioner` (v1alpha5) migrated to a `NodePool` (v1beta1):
+
+
+**Provisioner example (v1alpha)**
+
+```
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+ ...
+spec:
+  providerRef:
+    name: default
+  annotations:
+    custom-annotation: custom-value
+  labels:
+    team: team-a
+    custom-label: custom-value
+  requirements:
+  - key: karpenter.k8s.aws/instance-generation
+    operator: Gt
+    values: ["3"]
+  - key: karpenter.k8s.aws/instance-category
+    operator: In
+    values: ["c", "m", "r"]
+  - key: karpenter.sh/capacity-type
+    operator: In
+    values: ["spot"]
+  taints:
+  - key: example.com/special-taint
+    value: "true"
+    effect: NoSchedule
+  startupTaints:
+  - key: example.com/another-taint
+    value: "true"
+    effect: NoSchedule
+  kubeletConfiguration:
+    systemReserved:
+      cpu: 100m
+      memory: 100Mi
+      ephemeral-storage: 1Gi
+    maxPods: 20
+  limits:
+    resources:
+      cpu: 1000
+      memory: 1000Gi
+  weight: 50
+```
+
+**NodePool example (v1beta1)**
+
+```
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+...
+spec:
+  template:
+    metadata:
+      annotations:
+        custom-annotation: custom-value
+      labels:
+        team: team-a
+        custom-label: custom-value
+    spec:
+      requirements:
+      - key: karpenter.k8s.aws/instance-generation
+        operator: Gt
+        values: ["3"]
+      - key: karpenter.k8s.aws/instance-category
+        operator: In
+        values: ["c", "m", "r"]
+      - key: karpenter.sh/capacity-type
+        operator: In
+        values: ["spot"]
+      taints:
+      - key: example.com/special-taint
+        value: "true"
+        effect: NoSchedule
+      startupTaints:
+      - key: example.com/another-taint
+        value: "true"
+        effect: NoSchedule
+      kubeletConfiguration:
+        systemReserved:
+          cpu: 100m
+          memory: 100Mi
+          ephemeral-storage: 1Gi
+        maxPods: 20
+  limits:
+    cpu: 1000
+    memory: 1000Gi
+  weight: 50
+```
```

Comment:
Do we want to provide any detail to describe the difference here. Something like, "Note that the `Limits` and `Weight` fields sit outside of the template section. The `Labels` and `Annotations` fields from the Provisioner are now under the `spec.template.metadata` section. Note that all other fields including requirements, taints, kubeletConfiguration, etc. are specified under the `spec.template.spec` section."

Reply:
Done.